src/test/java/be/seeseemelk/mockbukkit/WorldMockTest.java

```diff
@@ -1461,23 +1461,6 @@ void getHighestBlockYAt_Water()
 		assertEquals(20, worldMock.getHighestBlockYAt(2, 3, HeightMap.MOTION_BLOCKING));
 	}
 
-	@Test
-	void getHighestBlockYAt_Waterlogged()
```

Comment:
Why was this test removed?

Reply:
Long story short, I automatically generated the data needed for isSolid, and noticed that the manually data was invalid, therefore invalidating some tests. So to say, I could not find a waterlog-able material which already is not considered solid, which is required for that specific test